src/style-util.js

```diff
@@ -326,7 +326,11 @@ const CSS_BUILD_ATTR = 'css-build';
  */
 export function getCssBuild(element) {
   if (element.__cssBuild === undefined) {
-    element.__cssBuild = element.getAttribute(CSS_BUILD_ATTR);
+    const buildComment = getBuildComment(element);
```

Comment:
let's only do this work if we don't have an attribute value.

Reply:
Comment mode needs a test.

Reply:
Done and done